Makefile

```diff
@@ -6,21 +6,21 @@ help: Makefile
 
 ## test:      execute the automated test suite
 test:
-	pytest --cov=microhapulator --doctest-modules microhapulator/*.py
+	pytest --cov=microhapulator --doctest-modules microhapulator/*.py microhapulator/*/test_*.py
 
 ## devdeps:   install development dependencies
 devdeps:
-	pip3 install --upgrade pip setuptools
-	pip3 install wheel twine
-	pip3 install pycodestyle pytest-cov pytest-sugar
+	pip install --upgrade pip setuptools
+	pip install wheel twine
+	pip install pycodestyle pytest-cov pytest-sugar
```

Comment:
Using `pip3` is/should be the "correct" way but in practice it has created problems for me.